docs/changelog.rst

```diff
@@ -29,6 +29,7 @@ Fixed
 - Crash during generation of loosely-defined headers. `#621`_
 - Show exception information for test runs on invalid schemas with ``--validate-schema=false`` command-line option.
   Before the output sections for invalid endpoints were empty. `#622`_
+- Schema validation error on schemas containing numeric values in scientific notation without a dot. `#629`_
```

Comment:
Could you, please move it upper? This place is related to the `1.9.0` release - you can create a `Fixed` sub-section just after `Unreleased` section in the very top of this file

Reply:
It is around line 8